src/main/java/teammates/ui/webapi/RemindFeedbackSessionSubmissionAction.java

```diff
@@ -20,34 +22,63 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
         String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
 
-        FeedbackSessionAttributes feedbackSession = getNonNullFeedbackSession(feedbackSessionName, courseId);
+        if (isCourseMigrated(courseId) && isAccountMigrated(userInfo.getId())) {
```

Comment:
no need to check for both course and account regarding migration, you can assume that account is migrated first. The flow should be like this: check if account is migrated -> if yes, check if course is migrated; if no, course is not migrated, carry out the old logic

Reply:
Does this mean to swap the ordering of the check with Account migrated, then Course migrated?

Reply:
Fixed and resolved